admin/src/containers/Dashboard/StatsContext.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component, createContext } from 'react';
+import config from 'config';
+import { toast } from 'react-toastify';
+import PropTypes from 'prop-types';
+import internalFetch from 'helpers/internal-fetch';
+
+const StatsContext = createContext({
+  isLoading: false,
+  stats: {},
+  fetchStats: () => {}
+});
+
+export class StatsProvider extends Component {
```

Comment:
Do we need a context for fetching data without interaction?

Reply:
👌